frontend/server/src/Controllers/Contest.php

```diff
@@ -2776,12 +2776,13 @@ private static function escapeCsv($csvRow) {
     public static function apiDownload(\OmegaUp\Request $r) {
         $r->ensureIdentity();
 
-        $contest = self::validateStats($r['contest_alias'], $r->identity);
-
-        include_once 'libs/third_party/ZipStream.php';
-        $zip = new \ZipStream("{$r['contest_alias']}.zip");
-        \OmegaUp\Controllers\Problemset::downloadRuns($contest->problemset_id, $zip);
-        $zip->finish();
+        $contest = self::validateStats(strval($r['contest_alias']), $r->identity);
+        if (!is_null($contest->problemset_id)) {
```

Comment:
aquí más bien hay que hacer

```suggestion
        if (is_null($contest->problemset_id)) {
            throw new ...;
        }
        // el resto
```

Reply:
Listo